packages/theming/theme-tokens/src/getMacOSAliasTokens.ts

```diff
@@ -0,0 +1,27 @@
+import * as macOSLightAliasTokens from './light-macos/tokens-alias';
+import * as macOSDarkAliasTokens from './dark-macos/tokens-alias';
+import * as macOSLightHCAliasTokens from './light-high-contrast-macos/tokens-alias';
+import * as macOSDarkHCAliasTokens from './dark-high-contrast-macos/tokens-alias';
+import * as hcAliasTokens from './highContrast/tokens-alias';
+import { AppearanceOptions } from '@fluentui-react-native/theme-types';
+import { assertNever } from 'assert-never';
+
+export function getMacOSAliasTokens(mode: AppearanceOptions, isHighContrast: boolean) {
+  if (mode === 'light') {
+    if (isHighContrast) {
+      return macOSLightHCAliasTokens.default;
+    } else {
+      return macOSLightAliasTokens.default;
+    }
+  } else if (mode === 'dark') {
+    if (isHighContrast) {
+      return macOSDarkHCAliasTokens.default;
+    } else {
+      return macOSDarkAliasTokens.default;
+    }
+  } else if (mode === 'highContrast') {
+    return hcAliasTokens.default;
+  } else {
+    assertNever(mode);
+  }
```

Comment:
If we never want to hit the case `(mode === 'highContrast')`, then we could call `assertNever(mode)` for that case instead.

Nit: I wonder if we can consolidate the AppearanceOption mode highContrast / getCurrentAppearance with our new isHighContrast boolean...

Reply:
I think `assertNever` is meant to check for exhaustiveness. The appropriate check for catching that error should be `throw new Error`

Reply:
Yeah `assertNever` is there to check for case exhaustiveness. I don't think it would be bad to add code to the highContrast case to throw an error or something, since MacOS doesn't expect to hit it.